WalletWasabi.Tests/UnitTests/UserInterfaceTest/PocketSelectionTests.cs

```diff
@@ -850,6 +850,11 @@ public void IsOtherSelectionPossibleCases()
 		var output = selection.AutoSelectPockets(recipient);
 		Assert.True(selection.IsOtherSelectionPossible(output.SelectMany(x => x.Coins), recipient));
 
+		// No other pocket can be used case.
+		recipient = "Dan, Lucas";
```

Comment:
The recipient should be different than `Dan` or `Lucas` to cover this case `if (!remainingUsablePockets.Any()) `.

```suggestion
		recipient = "Adam";
```